app/views/katello/api/v2/products/base.json.rabl

```diff
@@ -7,13 +7,30 @@ extends 'katello/api/v2/common/org_reference'
 
 attributes :provider_id
 attributes :sync_plan_id
-attributes :sync_status
 attributes :sync_summary
 attributes :gpg_key_id
 attributes :redhat? => :redhat
 
 attributes :available_content => :available_content, :if => params[:include_available_content]
 
+node :sync_status do |product|
```

Comment:
oh right, that function isn't returning a presenter object, its just returning a hash.